contracts/token/ERC721/extensions/draft-ERC721Votes.sol

```diff
@@ -284,6 +284,11 @@ abstract contract ERC721Votes is ERC721, EIP712 {
         return _domainSeparatorV4();
     }
 
+    /**
+    * @dev Returns token voting power
+    */
+    function _getVotingPower(uint tokenId) internal virtual returns(uint256) {}
```

Comment:
This will default to 0! We need to add `return 1;` as a default.

Reply:
Solved on 29061447c0e3372d5cd3e8b7ff918f34d918e8e2